poetry.lock

```diff
@@ -444,6 +444,32 @@ version = "1.10.1"
 [package.dependencies]
 pytest = ">=2.7"
 
+[[package]]
+category = "main"
+description = "Python HTTP for Humans."
+name = "requests"
```

Comment:
I thought `requests` had already been introduced as a dependency?

Reply:
Maybe this locks it also as a dependency of some other package?

Reply:
yes it should already be installed as part of the index PR. Not sure why it's here. It could have moved position in the file as there were conflicts.

Reply:
It appears that `requests` is in `poetry.lock` file twice for some reason. However, I tried to remove one and Poetry fails to read the file... I'll leave it in for now.

Reply:
I found the issue and have removed the duplicate entries for requests and requests-mock.